tsconfig.json

```diff
@@ -11,11 +11,16 @@
         "moduleResolution": "node",
         "strictNullChecks": true,
         "importHelpers": true,
+        "allowSyntheticDefaultImports": true,
         "lib": [
             "DOM",
             "ES5",
             "ES6",
             "ES7"
+        ],
+        "typeRoots": [
+          "src",
```

Comment:
Why did you add `src`. I have no problems finding the types in `externalLibTypes.d.ts` without this setting.

Reply:
Ah, I didn't know that it would work without it as well and I didn't test it.

I'll try it out and remove it if it works.